docs/user/contacts.md

```diff
@@ -0,0 +1,4 @@
+# Contacts
+
+The preferred way of contacting the Vald team is to the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) or the email at vald@vdaas.org.
```

Comment:
I suggest writing some introduction first. something like 
```
If you are interested in Vald, or you have any question about Vald, please contact us.
Slack:   ....
E-mail: ....
```

Reply:
Thank you for your review.
I fix soon.